ansible/playbook.yml

```diff
@@ -88,4 +88,18 @@
           net.bridge.bridge-nf-call-iptables = 1
 
     - name: CNI - Tune iptables - run
-      command: sysctl --system
\ No newline at end of file
+      command: sysctl --system
+
+    - name: Install Docker SDK for Python
+      pip:
+        name: docker
+
+    - name: Checking for entry VAULT_ADDR env in .bashrc
+      shell: cat /home/vagrant/.bashrc | grep export VAULT_ADDR | wc -l
```

Comment:
```suggestion
      shell: cat /home/vagrant/.bashrc | grep "export VAULT_ADDR" | wc -l
```
Doesn't the grep part need quotes?
![image](https://user-images.githubusercontent.com/51820995/85552074-eaa44980-b622-11ea-9830-7488891ea789.png)

If not the word-count will always be 0, because the grep fails.